eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/UtxosSpec.hs

```diff
@@ -469,54 +453,41 @@ govPolicySpec ::
   SpecWith (ImpTestState era)
 govPolicySpec = do
   describe "Gov policy scripts" $ do
-    it "failing native script govPolicy" $ do
+    -- These tests rely on the script in the constitution, but we can only change the constitution after bootstrap.
```

Comment:
This is only a problem because we don't have a guardrail script in the genesis file, which we should change. I've created a ticket about it #4726